### PR TITLE
Remove devdependency uuid

### PR DESCRIPTION
**Pull request by torresgol10**
I have deleted the dependency and its type. This dependency is not used anywhere in the package/next.

### CODE REVIEW

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: 49e5ec52539c7b7a2f843f41b1bdb1bdca1e16c8

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

**Review by eps1lon — APPROVED** [review]
Makes sense, thank you!

**Comment by ijjk** [comment]
## Failing test suites 
Commit: 49e5ec52539c7b7a2f843f41b1bdb1bdca1e16c8

`pnpm test-dev test/e2e/app-dir/actions/app-action.test.ts`
- app-dir action handling > fetch actions > should handle redirects to routes that provide an invalid RSC response

<details>
<summary>Expand output</summary>

  ● app-dir action handling › fetch actions › should handle redirects to routes that provide an invalid RSC response

    expect(received).toContain(expected) // indexOf

    Expected substring: "Hello from a pages route"
    Received string:    "0.07200802321880695
    Client
    Server
    Client and Server
    0
    +1+1 (Slow)-1*2
    redirect to a pages route
    submit
    test"

      985 |
      986 |       await retry(async () => {
    > 987 |         expect(await browser.elementByCss('body').text()).toContain(
          |                                                           ^
      988 |           'Hello from a pages route'
      989 |         )
      990 |         expect(await browser.url()).toBe(`${next.url}/pages-dir`)

      at toContain (e2e/app-dir/actions/app-action.test.ts:987:59)
      at retry (lib/next-test-utils.ts:787:14)
      at Object.<anonymous> (e2e/app-dir/actions/app-action.test.ts:986:7)

</details>


Read more about building and testing Next.js in [contributing.md](https://github.com/vercel/next.js/blob/canary/contributing.md).

File: packages/next/src/compiled/sass-loader/cjs.js
Comment (eps1lon) [inline packages/next/src/compiled/sass-loader/cjs.js:1]: Only changes to module ids

**Pull request merged**